src/types.ts

```diff
@@ -93,6 +105,11 @@ export interface OptionsFormatters {
    */
   prettierOptions?: VendoredPrettierOptions
 
+  /**
+   * Custom overrides let you have different configuration for certain file extensions and specific files.
+   */
+  prettierOverrides?: (options: PrettierOverridesOptions) => TypedFlatConfigItem[]
```

Comment:
I am not sure if I understand the purpose of this hook. I'd say we better move it to a separate PR so we can move the SVG support forward faster

Reply:
OK, I will create a new PR for `prettierOverrides`.